### ISSUE TITLE
docstrings in salt/utils/network.py  are incorrect

### ISSUE THREAD

**Issue by sjorge** [saltstack/salt#29231]
Lots of ```salt '*' network.xxx``` but those don't seem to work at all. Makes sense since this is not a module.
```
[root@core ~]# /opt/local/salt/bin/salt-call --local network.host_to_ip example.com
[WARNING ] The tools 'dmidecode' and 'lspci' failed to execute because they do not exist on the system of the user running this instance or the user does not have the necessary permissions to execute them. Grains output might not be accurate.
[WARNING ] The tools 'dmidecode' and 'lspci' failed to execute because they do not exist on the system of the user running this instance or the user does not have the necessary permissions to execute them. Grains output might not be accurate.
[INFO    ] Executing command 'pkg_info -Q LOCALBASE pkgin' in directory '/root'
[INFO    ] Executing command '/usr/sbin/zfs -?' in directory '/root'
'network.host_to_ip' is not available.
```

**Comment by jfindlay** [comment]
Thanks for fixing this, @sjorge.

**Comment by cachedout** [comment]
Closed via #29237
